### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by YuliiaSaliuta**

### CODE REVIEW

**Review by vshvanska — APPROVED** [review]

File: cinema/models.py [hunk 1]
```diff
@@ -35,12 +39,19 @@ def full_name(self):
         return f"{self.first_name} {self.last_name}"
 
 
+def movie_image_file_path(instance, filename):
+    _, ext = os.path.splitext(filename)
```
Comment (NEkropo1) [inline cinema/models.py]: better to use more descriptive(full) names

File: cinema/serializers.py [hunk 2]
```diff
@@ -79,6 +104,7 @@ class Meta:
             "cinema_hall_name",
             "cinema_hall_capacity",
             "tickets_available",
+            "movie_image",
```
Comment (NEkropo1) [inline cinema/serializers.py]: not obligatory, but more logical would be to place it under movie_title.

File: media/uploads/movies/inception-9e718d7a-6b79-45e1-b5ec-9341075a62a7.jpg
Comment (NEkropo1) [inline media/uploads/movies/inception-9e718d7a-6b79-45e1-b5ec-9341075a62a7.jpg:1]: don't think we need this file

File: user/serializers.py [hunk 3]
```diff
@@ -22,3 +24,38 @@ def update(self, instance, validated_data):
...
+        if email and password:
+            user = authenticate(email=email, password=password)
+
+            if user:
+                if not user.is_active:
+                    msg = _("User account is disabled.")
+                    raise serializers.ValidationError(
+                        msg,
+                        code="authorization"
+                    )
```
Comment (NEkropo1) [inline user/serializers.py]: `django.contib.auth.authenticate` won't authenticate disabled users, so this check is redundant.

**Review by NEkropo1 — CHANGES_REQUESTED** [review]

**Review by NEkropo1 — APPROVED** [review]
LGTM!

**Pull request closed without merging**